packages/edit-post/package.json

```
@@ -1,6 +1,6 @@
 {
 	"name": "@wordpress/edit-post",
-	"version": "3.1.4",
+	"version": "3.1.5",
```

```suggestion
	"version": "3.1.4",
```

The versions are actually changed automatically when we publish the packages. So we should keep it as is and only update the change log for now